content/doc/upgrade-guide/2.121.adoc

```diff
@@ -0,0 +1,68 @@
+---
+layout: documentation
+title:  Jenkins Upgrade Guide
+notitle: true
+---
+
+== Upgrading to Jenkins LTS 2.121.x
+
+Each section covers the upgrade from the previous LTS release, the section on 2.121.1 covers the upgrade from 2.107.3.
+
+=== Upgrading to Jenkins LTS 2.121.1
+
+==== Remoting update
+
+https://github.com/jenkinsci/remoting/blob/master/CHANGELOG.md#320[Remoting changelog],
+https://issues.jenkins-ci.org/browse/JENKINS-50237[JENKINS-50237]
+
+The remoting library has been updated to improve diagnosability and reliability.
+
+One of those changes, JENKINS-50237, requires that the `agent.jar` / `slave.jar` on agents is updated as well to be effective.
```

Comment:
as well as other packaging types like Swarm Plugin Client or Docker images.
Would worth mentioning.

Reply:
Could you elaborate? It's not clear to me what exactly you mean.